sufia.gemspec

```diff
@@ -41,6 +41,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'activerecord-import', '~> 0.5'
   spec.add_dependency 'posix-spawn'
   spec.add_dependency 'kaminari_route_prefix'
+  spec.add_dependency 'active-fedora', '= 9.9'
```

Comment:
Do you need to pin `active-fedora`? Couldn't you pin `activefedora-aggregations` instead?

Or, is it even worth pinning anything? If you're updating dependencies, why would you selectively update `sufia` and not `aggregations`?